### PR TITLE
[IMP] website: redesign `s_company_team` snippet

### PR DESCRIPTION
**Pull request by stefanorigano**
This PR is handled by @anso-odoo 

---

The goal of this commit is to improve the visual design of the default `s_company_team` snippet.

task-3657640

Requires:
- https://github.com/odoo/design-themes/pull/844

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by anso-odoo** [comment]
@stefanorigano I think you can have a look at this one. Enterprise is red but it doesn't look related to snippets at all...

**Review by stefanorigano — COMMENTED** [review]
Hello @anso-odoo , just a very fast pass.
If possible, I'd prefer if the background behind the person it's the same for each image. It would be better suit in case of corporate websites (and doesn't hurt  for other business as well ;)).
Ideally the background color shouldn't be pure white, so we avoid that the picture may blend with the snippet's background.

Eg, this is good ⬇︎
image


 Sidenote: fyi not sure if it's intentional, but you're replacing  `s_team_member_3.jpg` with the old `s_team_member_4.jpg`

image

**Comment by anso-odoo** [comment]
@stefanorigano good point, i changed with better images with same bg :)

File: addons/website/views/snippets/s_company_team.xml [hunk 1]
```diff
@@ -20,35 +23,38 @@
                         </div>
                     </div>
                 </div>
-                <div class="col-lg-6 pt24 pb24">
+                <div class="col-lg-6 pt32 pb32">
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_company_team.xml:26]: Let's give this guy a real `name` ;)

```suggestion
                <div class="col-lg-6 pt32 pb32" data-name="Member">
```
| before | after |
|--------|--------|
| image | image |

File: addons/website/static/src/snippets/s_company_team/000.scss [hunk 2]
```diff
@@ -2,7 +2,7 @@
 .s_company_team {
     @include media-breakpoint-down(lg) {
         img {
-            max-width: 50%;
+            height: 128px;
```
Comment (stefanorigano) [inline addons/website/static/src/snippets/s_company_team/000.scss:5]: Looks very arbitrary. Could we at least use `rem` units?

File: addons/website/views/snippets/s_company_team.xml [hunk 3]
```diff
@@ -4,14 +4,17 @@
...
+            <h3>Meet our team</h3>
+            <p class="lead">Dedicated professionals driving our success</p>
             <div class="row s_nb_column_fixed">
-                <div class="col-lg-6 pt24 pb24">
+                <div class="col-lg-6 pt32 pb32">
                     <div class="row s_col_no_resize s_col_no_bgcolor">
-                        <div class="col-lg-4 pb16 o_not_editable" contenteditable="false">
-                            <img alt="" src="/web/image/website.s_company_team_image_1" class="img-fluid rounded-circle mx-auto o_editable_media"/>
+                        <div class="col-lg-3 pb24 o_not_editable" contenteditable="false">
+                            <img alt="" src="/web/image/website.s_company_team_image_1" class="img-fluid w-lg-100 mx-lg-auto rounded-circle o_editable_media"/>
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_company_team.xml:13]: nitpicking: let's not enforce the classes order convention on lines that are just partially modified, it makes reading the diff a bit more complicated ;)

**Review by stefanorigano — COMMENTED** [review]
Hello @anso-odoo , nice job here, the edition process is neat 👍 
Just a few nitpicking comments that could be addressed during tech. review.

If my comment about `s_company_team_options` is confirmed (meaning that the option should be removed), then feel free to undraft and set this ready for tech. review.

Thanks 👍 

edit: @anso-odoo , actually I see that runbot is 🔴 again.. let's ensure that everything is fixed before sending to review ;)

File: addons/website/views/snippets/s_company_team.xml [hunk 4]
```diff
@@ -58,6 +64,17 @@
...
+    <xpath expr="//div[@data-js='Box']" position="inside">
+        <div data-selector=".s_company_team_member">
+            <we-colorpicker string="Card background" data-name="team_card_bg_opt"
+                data-select-style="true"
+                data-css-property="background-color"
+                data-color-prefix="bg-"/>
+        </div>
+    </xpath>
+</template>
+
```
[the remark this answers is not in the document]
Comment (anso-odoo) [reply addons/website/views/snippets/s_company_team.xml:77]: yes it's a leftover, mb

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

**Review by bvr-odoo — APPROVED** [review]
good work, thanks :+1:

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**